src/app/Caches/CreateOrUpdateEntryForm.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useState } from 'react';
+import React, {useEffect, useState} from 'react';
```

Comment:
revert? the other imports have a space between `{` and the "name". Other files also have a space in the imports.